ceph.spec.in

```diff
@@ -562,18 +562,60 @@ mkdir -p $RPM_BUILD_ROOT%{_localstatedir}/log/radosgw
 %clean
 rm -rf $RPM_BUILD_ROOT
 
+%pre
+%if 0%{?_with_systemd}
+  %if %{defined suse_version}
+    # service_add_pre and friends don't work with parameterized systemd service
+    # instances, only with single services or targets, so we always pass
+    # ceph.target to these macros (this and related comments all pertain to
+    # bnc#916499)
+    %service_add_pre ceph.target
+  %endif
+%endif
+
+
 %post
 /sbin/ldconfig
-/sbin/chkconfig --add ceph
+%if 0%{?_with_systemd}
+  systemd-tmpfiles --create %{_tmpfilesdir}/%{name}.conf
```

Comment:
The RPM should own the directory in question, so there's no need to run `systemd-tmpfiles --create` during `%post`. RPM can create the directory itself.